templates/default/sql/dml/1.3.0.sql.erb

```diff
@@ -11,3 +11,10 @@ DELETE FROM `hopsworks`.`variables` WHERE `id`='hiveserver_ext_hostname';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='hiveserver_ssl_hostname';
 REPLACE INTO `hopsworks`.`variables` VALUES ("hopsworks_enterprise", "<%= node['install']['enterprise']['install']%>");
 REPLACE INTO `hopsworks`.`variables` VALUES ("kubernetes_installed", "<%= node['install']['kubernetes'] %>");
+INSERT INTO `hopsworks`.`variables` VALUES ("yarn_runtime", "docker");
+INSERT INTO `hopsworks`.`variables` VALUES ("docker_mounts", "<%= node['hops']['docker']['mounts'] %>");
+REPLACE INTO `hopsworks`.`variables` VALUES ("jupyter_user", "<%= node['hops']['yarnapp']['user'] %>");
+REPLACE INTO `hopsworks`.`variables` VALUES ("jupyter_group", "<%= node['hops']['group'] %>");
+REPLACE INTO variables VALUES ("anaconda_dir", "/srv/hops/anaconda");
```

Comment:
You should build the path based on the node attributes

Reply:
can you use the full name `hopswork`.`variables` here as well?

Reply:
fixed